app/controllers/search_controller.rb

```diff
@@ -71,7 +78,8 @@ def forward_pattern_search(type, pattern, special_params)
          :observation, :project, :species_list, :user
       redirect_to_search_or_index(
         pattern: pattern,
-        search_path: send("#{type.to_s.pluralize}_path", pattern: pattern),
+        search_path: send("#{type.to_s.pluralize}_path",
+                          params: { pattern: pattern }.merge(special_params)),
         index_path: send("#{type.to_s.pluralize}_path", special_params)
```

Comment:
As of now, `special_params` for the `search_path` will only add a param if it's coming from obs_needing_ids.